src/core/wallets/btc/keystone/provider.ts

```diff
@@ -8,12 +8,14 @@ import { tapleafHash } from "bitcoinjs-lib/src/payments/bip341";
 import { toXOnly } from "bitcoinjs-lib/src/psbt/bip371";
 import { pubkeyInScript } from "bitcoinjs-lib/src/psbt/psbtutils";
 
-import type { BTCConfig, Fees, InscriptionIdentifier, UTXO } from "@/core/types";
+import type { BTCConfig, InscriptionIdentifier } from "@/core/types";
 import { Network } from "@/core/types";
 import BIP322 from "@/core/utils/bip322";
 import { toNetwork } from "@/core/utils/wallet";
 import { BTCProvider } from "@/core/wallets/btc/BTCProvider";
 
+import logo from "./logo.svg";
+
 initEccLib(ecc);
```

Comment:
We do this for every wallet

Reply:
> We do this for every wallet
I thought you are asking about
`import logo from "./logo.svg";`

> this initEccLib should be done inside the btc-staking-ts lib once only. should have nothing to do with wallets.
Do we think that `btc-staking-ts` is a part of `bbn-wallet-connect` BTC wallet provider (keyston in this example)? Or do we put only needed stuff there - in this case it's `initEccLib`

> we should remove it from wallet repo and check how it goes. IMO nothing will happen
it won't work, since Keystone is relying heavily on manual implementation of all the methods

Reply:
@jrwbabylonlab #128